iot_verifier/src/poc.rs

```diff
@@ -38,6 +39,15 @@ const POC_CELL_DISTANCE_MINIMUM: u32 = 8;
 /// the resolution at which parent cell distance is derived
 const POC_CELL_PARENT_RES: u8 = 11;
 
+lazy_static! {
+    /// Scaling factor when inactive gateway is not found in the tx scaling map (20%).
+    /// A default tx scale is required to allow for inactive hotspots to become active
+    /// again when an inactive hotspot's h3 index would otherwise be garbage-collected
+    /// from density scaling calculations and not finding a value on subsequent lookups
+    /// would disqualify the hotspot from validating further beacons
+    static ref DEFAULT_TX_SCALE: Decimal = Decimal::new(2000, 4);
```

Comment:
Just confirming that this value is `0.2000`

Reply:
according to the `rust_decimal` docs https://docs.rs/rust_decimal/latest/rust_decimal/struct.Decimal.html#method.new this should equate to
```
assert_eq!("0.200", Decimal::new(2000, 4).to_string())
```